README.md

```diff
@@ -0,0 +1,176 @@
+ Architecture: amd64
+ Description: Radeon Open Compute (ROCm) Runtime software stack
+ Homepage: https://github.com/RadeonOpenCompute/ROCm
+ Maintainer: ROCm Dev Support <rocm-dev.support@amd.com>
+ Package: rocm-core
+ Priority: optional
+ Section: devel
+ Version: 6.4.0.60400-crdnnh.15158~22.04
+ Installed-Size: 125
+
+$ dpkg -c rocm-core_6.4.0.60400-crdnnh.15158~22.04_amd64.deb
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/.info/
+-rw-r--r-- root/root        12 2024-11-22 10:37 ./opt/rocm-6.4.0/.info/version
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/include/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/include/rocm-core/
+-rw-r--r-- root/root      2801 2024-11-22 10:37 ./opt/rocm-6.4.0/include/rocm-core/rocm_getpath.h
+-rw-r--r-- root/root      2440 2024-11-22 10:37 ./opt/rocm-6.4.0/include/rocm-core/rocm_version.h
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/cmake/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/cmake/rocm-core/
+-rw-r--r-- root/root      2878 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/cmake/rocm-core/rocm-core-config-version.cmake
+-rw-r--r-- root/root      1590 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/cmake/rocm-core/rocm-core-config.cmake
+-rw-r--r-- root/root       842 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/cmake/rocm-core/rocmCoreTargets-release.cmake
+-rw-r--r-- root/root      3774 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/cmake/rocm-core/rocmCoreTargets.cmake
+lrwxrwxrwx root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/librocm-core.so -> librocm-core.so.1
+lrwxrwxrwx root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/librocm-core.so.1 -> librocm-core.so.1.0.60400
+-rw-r--r-- root/root     16640 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/librocm-core.so.1.0.60400
+-rw-r--r-- root/root       550 2024-11-22 10:37 ./opt/rocm-6.4.0/lib/rocmmod
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/libexec/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/libexec/rocm-core/
+-rw-r--r-- root/root      8208 2024-11-22 10:34 ./opt/rocm-6.4.0/libexec/rocm-core/runpath_to_rpath.py
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/share/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/share/doc/
+drwxr-xr-x root/root         0 2024-11-22 10:37 ./opt/rocm-6.4.0/share/doc/rocm-core/
+-rw-r--r-- root/root      1113 2024-11-22 10:34 ./opt/rocm-6.4.0/share/doc/rocm-core/LICENSE.txt
+
+
+```
+
+### Setting up locations
+
+By default, the install location is set to `/opt/rocm`. You can change this using
+`CMAKE_INSTALL_PREFIX`:
+
+```shell
+cmake -DCMAKE_INSTALL_PREFIX=<rocm-core-install-path> ..
+```
+
+## ROCM-CORE Library
+
+The flags for the lib would the following :
```

Comment:
I don't know what this is for, or why it is helpful.

Reply:
Thanks. This library dump is removed in latest version